utils/vlog/vlog_to_pbtype.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+
+"""\
+Convert a Verilog simulation model to a VPR `pb_type.xml`
+
+The following are allowed on a top level module:
+
+    - (* TYPE="bel|blackbox" *) : specify the type of the module
+    (either a Basic ELement or a blackbox named after the pb_type
+
+    - (* CLASS="lut|routing|flipflop|mem" *) : specify the class of an given
+    instance. Must be specified for BELs
+
+    - (* PB_NAME="name" *) : override the name of the pb_type
```

Comment:
Your adding this for when `.subckt` != the module name?

I think we can probably drop support for that?